dtfs-central-api/src/types/users.ts

```diff
@@ -0,0 +1,10 @@
+import { Team } from './tfm/teams';
+
+export type TfmUser = {
```

Comment:
I _think_ this represents something from the `tfm-users` collection? If so we should put it in `db-model/tfm-users.d.ts` and add `WithId`.